dist/common/sysconfig/scylla-server

```diff
@@ -42,3 +42,6 @@ SCYLLA_ARGS="--log-to-syslog 1 --log-to-stdout 0 --default-log-level info --netw
 
 # setup as AMI instance
 AMI=no
+
+# tune clocksource
+SET_CLOCKSOURCE=yes
```

Comment:
Why yes as default? It can cause timing to be off. We should only tune the clocksource if we're sure it's okay.

Reply:
The wording in scylla_setup should also reflect that it should only be done if the user knows what they are doing.